README.md

```diff
@@ -5,10 +5,6 @@ ci-sauce
 
 This folder contains the common code for the Bamboo and Jenkins Sauce OnDemand plugins.
 
-To update sauce connect, run the download script, src/main/resources/download.sh
```

Comment:
Yeah, but do we want to vendor the binaries?

Reply:
Yes, this could be added back when the SC5 latest version information is available.

Reply:
Oh, and vendoring the binaries in general is needed for the plugin so that it works on Jenkins instances that don't have access to the internet, we need to have them already on disk.